docs/src/SUMMARY.md

```diff
@@ -1,62 +1,78 @@
+    - [How to use thin-edge.io with your preferred init system](operate/configuration/019_how_to_use_preferred_init_system.md)
+    - [How to change temp path?](operate/configuration/018_update_config_paths.md)
+    - [Configuration management on child-devices](operate/configuration/child-device-config-management.md)
+  - [Security and Access Control](operate/security/index.md)
+      - [How to create a test certificate?](operate/security/003_registration.md)
+      - [How to add self-signed certificate root to trusted certificates list?](operate/security/010_add_self_signed_trusted.md)
+      - [How to set up client/server authentication for the local MQTT broker](operate/security/029_mqtt_local_broker_authentication.md)
+  - [Connection](operate/connection/index.md)
+    - [How to connect /index cloud end-point?](operate/connection/004_connect.md)
+  - [Processing Telemetry Data](operate/telemetry/index.md)
+    - [How to use `tedge mqtt` module?](operate/telemetry/005_pub_sub.md)
+  - [Operations](operate/operations/index.md)
+    - [How to restart your thin-edge.io device](operate/operations/016_restart_device_operation.md)
+  - [Monitoring](operate/monitoring/index.md)
+    - [How to enable watchdog using systemd?](operate/monitoring/021_enable_tedge_watchdog_using_systemd.md)
+    - [How to connect an external device?](operate/monitoring/013_connect_external_device.md)
+  - [Troubleshooting](operate/troubleshooting/index.md)
+    - [How to test the cloud connection?](operate/troubleshooting/007_test_connection.md)
+    - [How to access the logs on the device?](operate/troubleshooting/014_thin_edge_logs.md)
+    - [How to monitor health of tedge daemons](operate/troubleshooting/020_monitor_tedge_health.md)
+    - [How to trouble shoot device monitoring?](operate/troubleshooting/009_trouble_shooting_monitoring.md)
+  - [Device Management with Cumulocity](operate/c8y/index.md)
+    - [Supported Operations Management for Cumulocity IoT](operate/c8y/supported_operations.md)
+    - [How to add custom fragments to Cumulocity](operate/c8y/022_c8y_fragments.md)
+    - [How to retrieve logs with the log plugin](operate/c8y/023_c8y_log_plugin.md)
+    - [How to add C8Y SmartRest Templates](operate/c8y/024_smartrest_templates.md)
+    - [How to manage configuration files with Cumulocity](operate/c8y/025_config_management_plugin.md)
+    - [How to enable configuration management on child devices](operate/c8y/child_device_config_management_agent.md)
+    - [How to enable firmware management on child devices](operate/c8y/child_device_firmware_management.md)
+    - [How to remotely connect to your thin-edge.io device via SSH/VNC/Telnet with Cumulocity remote access](operate/c8y/027_remote_access_with_cumulocity.md)
+    - [How to monitor a service from Cumulocity](operate/c8y/028_c8y_service_monitoring.md)
+    - [How to manage Apama software artifacts with Apama plugin?](operate/c8y/017_apama_software_management_plugin.md)
```

Comment:
Again, as a nesting fan, I'd have kept this doc as a sub-level of the doc on software management ;)